.github/workflows/CI.yml

```diff
@@ -17,17 +17,8 @@ jobs:
         version:
           - '1.0'
           - '1'
-        os:
-          - ubuntu-latest
-          - macOS-latest
-          - windows-latest
-        arch:
-          - x86
-          - x64
-        # 32-bit Julia binaries are not available on macOS
-        exclude:
-          - os: macOS-latest
-            arch: x86
+        os: [ubuntu-latest]
+        arch: [x86, x64]
```

Comment:
Tempting to drop one of these.
Maybe from x64 since if it works on x86 it will work on x64.
and its likely that people have tested it on x64 locally.